src/lib/styles/forms.less

```diff
@@ -0,0 +1,50 @@
+/*************************
+ * From publish reset.css:
```

Comment:
I moved this stuff over from reset.css. Some of it is overwritten a lot of the time such as `vertical-align: bottom;` on checkbox and will be reviewed and updated separately with the rest of the forms stuff.